modin/core/execution/ray/implementations/pandas_on_ray/partitioning/partition.py

```diff
@@ -13,6 +13,7 @@
 
 """Module houses class that wraps data (block partition) and its metadata."""
 
+from modin.utils import Invocable
```

Comment:
please re-order the imports - we usually first import builtin and stdlib, then empty line, then from non-standard dependencies (like `pandas` or `ray`), then empty line, then Modin-defined packages

Reply:
done